### PR TITLE
feat(tool): Add fraction_input param to token limit model

### PR DESCRIPTION
**Pull request by klcd**
- Used to always have a default for limits on input and output tokens.

- CI/CD edits will be done if code is accepted

### CODE REVIEW

File: unique_toolkit/tests/language_model/test_language_models_infos.py [hunk 1]
```diff
@@ -63,3 +64,25 @@ def test_get_custom_language_model(self):
...
+        assert limits.token_limit_output == 6000.0
+
+    def test_language_model_token_limits_with_total_and_fraction(self):
+        limits = LanguageModelTokenLimits(token_limit=10000, fraction_input=0.2)
+        assert limits.token_limit_input == 2000.0
+        assert limits.token_limit_output == 8000.0
+
+    def test_language_model_token_limits_raises_error(self):
+        with pytest.raises(ValueError):
+            LanguageModelTokenLimits()
```
Comment (abiodun0) [inline unique_toolkit/tests/language_model/test_language_models_infos.py:88]: should add more tests where only `token_limit_input`  or `token_limit_output` is provided.

**Review by abiodun0 — APPROVED** [review]

File: unique_toolkit/unique_toolkit/language_model/schemas.py [hunk 2]
```diff
@@ -193,33 +193,32 @@ class LanguageModelStreamResponse(BaseModel):
 
 
 class LanguageModelTokenLimits(BaseModel):
-    token_limit: Optional[int] = None
     token_limit_input: Optional[int] = None
     token_limit_output: Optional[int] = None
+    token_limit: Optional[int] = None
```
Comment (sheiksadique) [inline unique_toolkit/unique_toolkit/language_model/schemas.py:198]: Looks like the order changed. Hope it isn't used anywhere without specifying the param name.

Comment (klcd) [reply unique_toolkit/unique_toolkit/language_model/schemas.py:198]: Initializing pydantic models is not possible afaik

Screenshot 2024-12-12 at 09 41 47

File: unique_toolkit/tests/language_model/test_language_models_infos.py (hunk 1 above)
Comment (klcd) [reply unique_toolkit/tests/language_model/test_language_models_infos.py:110]: Thanks, it's done

**Pull request merged**